components/dropbox/sources/all-updates/all-updates.mjs

```diff
@@ -1,25 +1,35 @@
-const dropbox = require("../../dropbox.app.js");
+import dropbox from "../../dropbox.app.mjs";
 
-module.exports = {
+export default {
   key: "dropbox-new-or-modified-file-or-folder",
   name: "New or Modified File or Folder",
```

Comment:
Key should be:
`key: "dropbox-all-updates",`

If you create a `common.mjs` file in sources, include it here
```
export default {
   ...common,
   key: "dropbox-all-updates",
```